mudlet-mapper.xml

```diff
@@ -8229,7 +8279,7 @@ end</script>
                     <Script isActive="yes" isFolder="no">
                         <name>register_achaeas_envdata</name>
                         <packageName></packageName>
-                        <script>function register_achaeas_envdata(_, game)
+                        <script>function register_achaeas_envdata(game)
```

Comment:
This change shouldn't actually be part of this PR, right?

Reply:
Yea that wasn't supposed to be there. I'll get that fixed up as soon as I can.